theme/base-2016/extrafields.twig

```diff
@@ -4,7 +4,7 @@
 
     <h1>{{ record.title }}</h1>
 
-    {{ fields(template = 'partials/_sub_fields.twig') }}
+    {{ fields(record = record, template = 'partials/_sub_fields.twig') }}
```

Comment:
I mean when the user doesn't pass record. i.e. The PHP code still tries to pull it from globals.